presto-testing/src/main/java/io/prestosql/testing/AbstractTestQueryFramework.java

```diff
@@ -306,7 +324,11 @@ protected void assertAccessDenied(
                 fail("Expected " + AccessDeniedException.class.getSimpleName());
             }
             catch (RuntimeException e) {
-                assertExceptionMessage(sql, e, ".*Access Denied: " + exceptionsMessageRegExp);
+                String query = "Query: " + sql;
+                assertThat(e).as(query).hasMessageMatching(exceptionsMessageRegExp);
+                if (exceptionsCauseMessageRegExp != null) {
```

Comment:
Actually we do not care about the cause, as this is not printed to the user. Please undo these changes.

Reply:
done